metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -1364,7 +1364,8 @@ Boolean isBrokerUnfenced(int brokerId) {
                 setErrorCode(apiError.error().code()).
                 setErrorMessage(apiError.message()));
         }
-        return new ControllerResult<>(records, results, true);
+        log.debug("CreatePartitions result(s): {}", results);
```

Comment:
Hmm.. It is useful to know in the logs if `validateOnly` was set.

Reply:
I'll put a debug log in the controller layer before processing the final list of topics.

Reply:
I can still see this causing confusion. Perhaps we can move it to `QuorumController` where we have `validateOnly` and can make the message clear?